packages/components/src/components/CodeBlock/CodeBlock.tsx

```
@@ -0,0 +1,354 @@
+      <pre ref={preRef} className={s.pre}>
+        <code
+          className={clsx(
+            'hljs',
+            s.code,
+            utilClasses.typography['mono-codeblock']
+          )}
+          data-language={pending || failed ? undefined : language}
+          // `highlight.js` escapes the raw source before wrapping tokens in spans — see useHighlightedCode.
+          dangerouslySetInnerHTML={
+            pending || failed ? undefined : { __html: html }
+          }
```

Biome (2.5.6)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtmlWithChildren)